### DISCUSSION TITLE
weird auto-assigning of tags and document types

### DISCUSSION THREAD

**Discussion by tcurdt** [jonaswinkler/paperless-ng#1103] [Support]
I have no matching specified (except for vat)

![Screen Shot 2021-06-06 at 15 04 33](https://user-images.githubusercontent.com/13697/120925587-78186680-c6d9-11eb-9c11-1f5d2865d53b.png)

yet on import I am getting rather useless tags set.
Why is that? Where is the matching coming from?

Same goes for document type

![Screen Shot 2021-06-06 at 15 13 07](https://user-images.githubusercontent.com/13697/120925638-bf9ef280-c6d9-11eb-820e-4e135db73d65.png)

**Comment by Pyromane** [comment]
Please show the "Edit" view of one of the problem tags.

**Reply by tcurdt** [reply]
This for example gets applied - and I don't see why:

![Screen Shot 2021-08-04 at 19 41 59](https://user-images.githubusercontent.com/13697/128228694-4716de5d-8a9a-4608-8966-890f9f96c174.png)
